src/domain/accounts/services/src/authentication_service_impl.rs

```diff
@@ -136,16 +129,50 @@ impl AuthenticationServiceImpl {
     pub fn decode_access_token(
```

Comment:
Method `make_access_token` is doing JWT token, a brain killing mess in terminology.